CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -228,6 +229,7 @@ void SiPixelLorentzAnglePCLHarvester::dqmEndJob(DQMStore::IBooker& iBooker, DQMS
   f1->SetParName(5, "quintic term");
 
   double p1_simul_newmodule = 0.294044;
+  double half_width = 0.0285 / 2 * 10000;  // pixel width in units of micro meter
```

Comment:
No magic numbers,  please fetch from geometry

Reply:
@wweiphy  can you please do this?